plugins/teststeps/cmd/cmd.go

```diff
@@ -80,14 +82,16 @@ func (ts *Cmd) Run(cancel, pause <-chan struct{}, ch test.TestStepChannels, para
 		}
 		cmd := exec.CommandContext(ctx, ts.executable, args...)
 		cmd.Dir = ts.dir
-		var stdout, stderr bytes.Buffer
-		cmd.Stdout, cmd.Stderr = &stdout, &stderr
+		var stdoutBuf, stderrBuf bytes.Buffer
+
+		cmd.Stdout = io.MultiWriter(os.Stdout, &stdoutBuf)
+		cmd.Stderr = io.MultiWriter(os.Stderr, &stderrBuf)
```

Comment:
It does not look like "Add working dir parameter". Could you elaborate why these lines are required?